pkgs/os-specific/darwin/yabai/default.nix

```diff
@@ -1,25 +1,19 @@
-{ lib, stdenv, fetchFromGitHub, darwin, xxd }:
+{ lib, stdenv }:
+
+ /*
+ This cannot be built from source as it requires entitlements and
+ for that it needs to be code signed.
+ */
 
 stdenv.mkDerivation rec {
   pname = "yabai";
-  version = "3.3.10";
+  version = "4.0.2";
 
-  src = fetchFromGitHub {
-    owner = "koekeishiya";
-    repo = pname;
-    rev = "v${version}";
-    sha256 = "sha256-8O6//T894C32Pba3F2Z84Z6VWeCXlwml3xsXoIZGqL0=";
+  src = fetchTarball {
+    url = "https://github.com/koekeishiya/${pname}/releases/download/v${version}/${pname}-v${version}.tar.gz";
+    sha256 = "00nxzk1g0hd8jqd1r0aig6wdsbpk60551qxnvvqb9475i8qbzjf6";
   };
 
-  nativeBuildInputs = [ xxd ];
-
```

Comment:
We do not need to do any patching?

Reply:
From what I can see, yabai doesn't appear to have any problematic references to non-writeable nix directories or nix stores, and doesn't appear to have any dependencies that need to be referenced in the derivation.